### PR TITLE
prismlauncher: 7.1 -> 7.2

### PR DESCRIPTION
**Pull request by Scrumplex**
###### Description of changes

Minor release, no packaging changes.

https://github.com/PrismLauncher/PrismLauncher/releases/tag/7.2

https://github.com/PrismLauncher/PrismLauncher/compare/7.1...7.2

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/games/prismlauncher/default.nix [hunk 1]
```diff
@@ -46,7 +46,10 @@ stdenv.mkDerivation rec {
 
   hardeningEnable = [ "pie" ];
 
-  cmakeFlags = lib.optionals (msaClientID != null) [ "-DLauncher_MSA_CLIENT_ID=${msaClientID}" ]
+  cmakeFlags = [
+    # downstream branding
+    "-DLauncher_BUILD_PLATFORM=nixpkgs"
+  ] ++ lib.optionals (msaClientID != null) [ "-DLauncher_MSA_CLIENT_ID=${msaClientID}" ]
```
Comment (Minion3665) [inline pkgs/games/prismlauncher/default.nix:52]: can I query the

> no packaging changes

bit please? I think from the comment this is only doing branding but I want to make sure it's nothing more than that

**Review by Minion3665 — CHANGES_REQUESTED** [review]

File: pkgs/games/prismlauncher/default.nix (hunk 1 above)
Comment (Scrumplex) [reply pkgs/games/prismlauncher/default.nix:52]: Relevant pull request: https://github.com/PrismLauncher/PrismLauncher/pull/1276

Comment (Minion3665) [reply pkgs/games/prismlauncher/default.nix:52]: thanks, I think this is fine then, thank you very much!

**Review by Minion3665 — APPROVED** [review]

**Review by getchoo — APPROVED** [review]

**Review by getchoo — CHANGES_REQUESTED** [review]
almost forgot, the commit for libnbtplusplus needs to be bumped :p
https://github.com/PrismLauncher/PrismLauncher/pull/1419

**Comment by Scrumplex** [comment]
> almost forgot, the commit for libnbtplusplus needs to be bumped :p [PrismLauncher/PrismLauncher#1419](https://github.com/PrismLauncher/PrismLauncher/pull/1419)

that's 7.3

**Comment by getchoo** [comment]
oops, i thought it got merged into 7.2 :derp:

**Review by getchoo — APPROVED** [review]

**Comment by getchoo** [comment]
Result of `nixpkgs-review pr 245529` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>prismlauncher</li>
    <li>prismlauncher-qt5</li>
    <li>prismlauncher-qt5-unwrapped</li>
    <li>prismlauncher-unwrapped</li>
  </ul>
</details>

**Pull request merged**